.github/pull_request_template.md

```diff
@@ -1,51 +1,50 @@
+  List any https://mariadb.com/kb/ pages that need changing.
+-->
 
+## How to test this PR?
 TODO: modify the automated test suite to verify that the PR causes MariaDB to behave as intended.
-Consult the documentation on ["Writing good test cases"](https://mariadb.org/get-involved/getting-started-for-developers/writing-good-test-cases-mariadb-server).
 <!--
-In many cases, this will be as simple as modifying one `.test` and one `.result` file in the `mysql-test/` subdirectory.
-Without automated tests, future regressions in the expected behavior can't be automatically detected and verified.
+  Consult the documentation on ["Writing good test cases"](https://mariadb.org/get-involved/getting-started-for-developers/writing-good-test-cases-mariadb-server).
+  In many cases, this will be as simple as modifying one `.test` and one `.result` file in the `mysql-test/` subdirectory.
+  Without automated tests, future regressions in the expected behavior can't be automatically detected and verified.
 -->
+If the changes are not amenable to automated testing, please explain why and carefully describe how to test manually.
 
-If the changes are not amenable to automated testing, please explain why not and carefully describe how to test manually.
-
+## Basing the PR on the correct MariaDB version
 <!--
-Tick one of the following boxes [x] to help us understand if the base branch for the PR is correct.
+  Tick (`[x]`) one of the following boxes to help us understand if the base branch for the PR is correct.
 -->
-## Basing the PR against the correct MariaDB version
-- [ ] *This is a new feature or a refactoring, and the PR is based against the `main` branch.*
-- [ ] *This is a bug fix, and the PR is based against the earliest maintained branch in which the bug can be reproduced.*
+- [ ] *This is a new feature or a refactoring, and the PR is based on the `main` branch.*
+- [ ] *This is a bug fix, and the PR is based on the earliest maintained branch in which the bug can be reproduced.*
 
+## PR Quality Check
 <!--
   All code merged into the MariaDB codebase must meet a quality standard and coding style.
-  Maintainers are happy to point out inconsistencies but in order to speed up the review and merge process we ask you to check the CODING standards.
+  Maintainers are happy to point out inconsistencies, but to speed up the review and merge process, we ask you to verify the Coding Standards.
 -->
-## PR quality check
-- [ ] I checked the [CODING_STANDARDS.md](https://github.com/MariaDB/server/blob/-/CODING_STANDARDS.md) file and my PR conforms to this where appropriate.
-- [ ] For any trivial modifications to the PR, I am ok with the reviewer making the changes themselves.
+- [ ] I checked the [CODING_STANDARDS.md](https://github.com/MariaDB/server/blob/-/CODING_STANDARDS.md) file, and my PR conforms to this where appropriate.
+- [ ] For any trivial modifications to the PR, I am ok with the reviewer making the changes themselves.
```

Comment:
Somehow, the PR UI doesn’t strip the trailing ␊.
It’s standard for files, but looks odd in a multiline textbox.